SimPEG/Mesh/TensorMesh.py

```diff
@@ -217,7 +229,7 @@ def isInside(self, pts, locType='N'):
             inside = inside & (pts[:,i] >= tensor.min()-TOL) & (pts[:,i] <= tensor.max()+TOL)
         return inside
 
-    def getInterpolationMat(self, loc, locType='CC', zerosOutside=False):
+    def _getInterpolationMat(self, loc, locType='CC', zerosOutside=False):
```

Comment:
This is so I can call it from the CylMesh Interpolation. The Edge, CC and Face interpolation from Tensor Mesh are all valid for Cyl

![image](https://cloud.githubusercontent.com/assets/6361812/20459516/29762c42-ae79-11e6-9587-e23d71277a9d.png)